src/components/views/messages/ReactionsRow.tsx

```diff
@@ -194,13 +194,12 @@ export default class ReactionsRow extends React.PureComponent<IProps, IState> {
         let showAllButton;
         if ((items.length > MAX_ITEMS_WHEN_LIMITED + 1) && !showAll) {
             items = items.slice(0, MAX_ITEMS_WHEN_LIMITED);
-            showAllButton = <a
+            showAllButton = <button
```

Comment:
should we be using an AccessibleButton here? If not, can we get a comment to explain why this one is different?